test/specs/e2e_wiki.js

```diff
@@ -0,0 +1,52 @@
+
+import MainPage from '../pageobjects/main.page.js'
+import mainPage from "../pageobjects/main.page.js";
+import Pages from "../pageobjects/pages.js";
+
+describe('Wikipedia main page', () => {
+    beforeEach(() => {
+        MainPage.open();
+    });
+
+    after( () => {
+        browser.closeWindow();
+    })
+
+    it('should see Wikipedia page elements', async ()  => {
+        await expect(MainPage.logoWikipedia).toBeExisting();
+        await expect(MainPage.logoWikipedia).toHaveTextContaining("Wikipedia");
+        await expect(mainPage.fieldSearch).toBeExisting()
```

Comment:
You can update mainPage to `MainPage` after deleting the mainPage import statement above.